core/util/src/main/java/org/eclipse/rdf4j/common/iteration/IterationWrapper.java

```diff
@@ -38,7 +38,7 @@
 	 *
 	 * @param iter The wrapped Iteration for this <tt>IterationWrapper</tt>, must not be <tt>null</tt>.
 	 */
-	protected IterationWrapper(Iteration<? extends E, ? extends X> iter) {
+	public IterationWrapper(Iteration<? extends E, ? extends X> iter) {
```

Comment:
If you use IteratorCloseableIteration instead, you don't have to change visibility here.